README.md

```diff
@@ -48,11 +48,15 @@ We support both Python 2.7 and 3.5 or later. Some OS specific prerequisites belo
 
 #### Optional
 
-* If you wish to decorate your console log with all kinds of colors,
-install the coloredlogs module using pip. `pip install coloredlogs`
-    * There have been issues with coloredlogs installation on Windows.
-     We might switch to a different module at some point to enable
-     colored logging on Windows as well.
+There is punch of optional dependencies which brings some optional features,
```

Comment:
Typo, punch->bunch. Also very slangy way to say it, would be better to change into something more "standard".